parlai/agents/transformer/polyencoder.py

```diff
@@ -96,6 +95,29 @@ def add_cmdline_args(cls, argparser):
         )
         return agent
 
+    @classmethod
+    def upgrade_opt(cls, opt_from_disk: Opt):
+        # call the parent upgrades
+        opt_from_disk = super(PolyencoderAgent, cls).upgrade_opt(opt_from_disk)
+
+        polyencoder_attention_keys_value = opt_from_disk.get(
+            'polyencoder_attention_keys'
+        )
+        if polyencoder_attention_keys_value is not None:
+            # 2020-02-19 We are deprecating this flag because it was used for a one-time
+            # set of experiments and won't be used again. This flag was defaulted to
+            # 'context', so throw an exception otherwise.
+            if polyencoder_attention_keys_value == 'context':
+                warn_once(
```

Comment:
This is the "I'm loading from an old model and I didn't specify this behavior so I used the default" case. The user doesn't need to be warned that nothing changed here. Just silently delete the key from the opt and go on.

Reply:
Yes - changed